src/components/pages/Notes/NoteGroupNav.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import { connect } from "react-redux";
+import { values } from "lodash";
+import { NavLink, Route } from 'react-router-dom';
+
+import Button from "../../library/Button";
+
+
+function NoteGroupNav({ children, noteGroups, addGroup, deleteGroup, updateGroupName }) {
+  const [editedGroupId, setEditedGroupId] = useState();
+  const [editedGroupName, setEditedGroupName] = useState('jeanPaulBelmondo');
+
+  return (
+    <div className="group-nav">
+      <div className="columns">
+        <div className="column is-2">
+          <aside className="menu">
+            <ul className="menu-list">
+              {noteGroups.map(noteGroup => {
+                return(
+                    <li key={noteGroup.id}>
+                      {editedGroupId === noteGroup.id ? (
+                        <input
+                          className="input is-primary is-small"
+                          autoFocus
+                          value={editedGroupName}
+                          onChange={e => setEditedGroupName(e.target.value)}
+                        />
+                      ) : (
+                        <NavLink to={`/notes/noteGroups/${noteGroup.id}`} className="note-group">
+                          {noteGroup.name}
+                          <div className="actions">
```

Comment:
Don't forget to add `display: none` and `display: block` on hover. 
Also, in your sass you did:
```sass
.actions
    position: absolute
    left: 150px
```
Rather than positioning using `left`, in this case you prefer to position using `right` so that the buttons will always stick to the right of the note group.

Reply:
j'arrive pas :'( :'( si je met position right ils restent pas dans le container, ils partent tout à droite de l'écran

et display pareil i dont manage #tristesse

Reply:
Ok je vois. En fait il faut comprendre que quand tu donnes à un élément `position: absolute`, il se place par rapport au plus proche parent dans son arbre qui a lui aussi un `position` définie.
Donc là toi tu veux que les actions se placent tout à droite de leur menu item (qui est ici un lien note group). Donc il faut que tu ajoutes sur chaque menu item `position: relative`, qui ne change rien au positionnement de l'item en tant que tel mais qui va permettre aux éléments en dessous de se placer relativement à lui si besoin.

Personnellement dans mon truc j'ai fait ça:
```sass
    ...
    .menu
      height: 100%
      position: relative
      
      li > a
        display: block
        position: relative

        /* Pas relatif au propos mais bon à savoir: ça fait que le texte se raccourcie et met ... à la fin si il n'a pas la place nécessaire. */
        .name
          max-width: calc(100% - 17px)
          text-overflow: ellipsis
          overflow: hidden
          white-space: nowrap

        .actions
          display: none
          position: absolute
          right: 3px
          top: 3px

       /* Le character & avec sass sert à référencer le selecteur CSS courant. Donc ici je dis que quand menu li > a:hover, la classe `.actions` du dessous doit devenir `display: block` */
       &:hover
          .actions
            display: block
```

Reply:
Aah ok position relative j'avais pas en tête, j'ai changé mais comme on n'a pas le même sass je suis pas sure que ça soit bon. Chaude d'en reparler sur skype ^^